README.md

```diff
@@ -1 +1,33 @@
-# awesomeBooks
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awesome Books
+
+In this project, I built a basic website that allows users to add/remove books from a list. I achieved that by using JavaScript objects and arrays, I also dynamically modified the DOM and added basic events.
+
+## Built With
+
+- HTML
+- Javascript
+
+## [Demo link](https://mutalenic.github.io/awesomeBooks/)
```

Comment:
I think it's better to put the live demo link in a separate line and not with the title, here is a suggestion:

## Demo Link
[Live demo link](https://mutalenic.github.io/awesomeBooks/)